README.md

```diff
@@ -9,5 +9,12 @@ A Chat App for Adventures in Flutter.
 Get `GoogleService-Info.plist` and `google-services.json` from the [Shared Project Folder](https://drive.google.com/drive/folders/1URxuILxk8NR66DIRbuERFDXIILCX6jM2?usp=sharing)
 
 Add them to your project at:
+
 - `ios/Runner/GoogleService-Info.plist`
 - `android/app/google-services.json`
+
```

Comment:
Could we put a heading here to separate your addition from the "Missing Files" section?  Maybe "Common Commands" or something?